tests/data/nccm/data.py

```diff
@@ -24,15 +24,8 @@ def create_file(path: str, dtype: str):
         "driver": "GTiff",
         "dtype": dtype,
         "count": 1,
-        "crs": CRS.from_epsg(4326),
-        "transform": Affine(
-            8.983152841195208e-05,
-            0.0,
-            115.483402043364,
-            0.0,
-            -8.983152841195208e-05,
-            53.531397320113605,
-        ),
+        "crs": CRS.from_epsg(32616),
+        "transform": Affine(10, 0.0, 399960.0, 0.0, -10, 4500000.0),
```

Comment:
If you copy the CRS/transform from the existing S2 fake data, you no longer need to create new fake data for S2. So you can get rid of all the new files in this PR and only update existing files.